test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -988,10 +992,10 @@ public void testSnapshotSyncLongDurationApply() throws Exception {
      */
     @Test
     public void testSnapshotSyncDelayedApplyResponse() throws Exception {
-        testSnapshotSyncAndLogEntrySync(0, true);
+        testSnapshotSyncAndLogEntrySync(0, true, 0);
     }
 
-    private void testSnapshotSyncAndLogEntrySync(int numCyclesToDelayApply, boolean delayResponse) throws Exception {
+    private void testSnapshotSyncAndLogEntrySync(int numCyclesToDelayApply, boolean delayResponse, int dropAcksLevel) throws Exception {
```

Comment:
a little hesitant on "count" part since it may infer that it corresponds to number of ACKs dropped.

This variable is to decide if ACKs are to be randomly dropped OR if msgs have to be dropped after an ACK drop to emulate the Active's part in the testLogEntrySyncWithFSMChangeAndWithAckDrop scenario

Reply:
you're right, I then saw the meaning and forgot to remove the comment!